pyproject.toml

```diff
@@ -18,6 +18,7 @@ classifiers = [
     "License :: OSI Approved :: Apache Software License",
     "Programming Language :: Python :: 3",
 ]
+requires-python = ">=3.9"
```

Comment:
Ah got it. Yeah I think we'll need to downgrade for now. Given that this tool is internal, I don't think we need to make this CLI all that full-featured. I would be fine only defining `add_argument('--no-validate', action='store_false')` (assuming that we want to make validation opt-out rather than opt-in) and then switching to `BooleanOptionalAction` when we drop 3.8 support to have both `--validate` and `--no-validate` support.

Reply:
Done.